agensgraph/_property.py

```diff
@@ -0,0 +1,148 @@
+'''
+Copyright (c) 2014-2018, Bitnine Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+
+import sys
+
+from psycopg2.extensions import ISQLQuote
+from psycopg2.extras import json
+
+# borrowed from simplejson's compat.py
+if sys.version_info[0] < 3:
+    string_types = (basestring,)
+    integer_types = (int, long)
+    def dict_items(o):
+        return o.iteritems()
+else:
+    string_types = (str,)
+    integer_types = (int,)
+    def dict_items(o):
+        return o.items()
+
+def quote_string(s):
+    s = s[1:-1]
```

Comment:
Can you tell me why s[0] is excluded ? 
Is that something unnecessary character?

Reply:
`quote_string()` will take an argument that is an encoded value returned by `json.dumps()` if the original value is string type. That means, `quote_string()` will take a valid JSON string. However, AgensGraph that implements Cypher expression cannot accept that format as a valid string value. So, it have to be wrapped with `'` charcter instead of `"` character, and the line of code removes wrapping `"` character first.